src/OpenTracing/Standard/Format.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OpenTracing\Standard;
+
+class Format {
+
+	const SPLIT_TEXT = 'text';
+	const SPLIT_BINARY = 'binary';
+	const PACKED_HTTP_HEADERS = 'packed_http_headers';
+	const RAW_HTTP_HEADERS = 'raw_http_headers';
```

Comment:
it doesn't look like these two formats should be in the official Formats enum. The reason we had one in Go was because Go has a widely used http.Header type in standard library. But these two formats do not introduce a new intermediate representation, they are simply built on top of split text representation.

The can be provided as util methods. Things placed in the standard Format enum require all tracers to implement them, which is not necessary here.